qcoro/qcoro.h

```diff
@@ -4,7 +4,11 @@
 
 #pragma once
 
+#include <qglobal.h>
+
 #include "task.h"
 #include "qcorocore.h"
-#include "qcorodbus.h"
+#if defined(QT_DBUS_LIB)
+#  include "qcorodbus.h"
+#endif
```

Comment:
This should probably take into account `QCORO_WITH_QTDBUS` and, since in theory this will make the header unusable when QCoro is built without DBus, even if Qt has DBus support.

Reply:
Yes - you are right. Adding this to cmake shouldn't be a big deal though: just check for `if (TARGET Qt::DBus)`